lintro/ai/review/progress.py

```
@@ -184,6 +215,18 @@ def on_abort(self) -> None:
         """Stop the bar without printing a completion summary."""
         self._stop_progress()
 
+    def on_error(
+        self,
+        *,
+        chunk_index: int | None = None,
```

Ruff (0.15.20)

[warning] Unused method argument: `chunk_index`

(ARG002)